core/matching/matchers/jwt_match.go

```diff
@@ -0,0 +1,59 @@
+package matchers
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+)
+
+var JWT = "jwt"
+
+func JwtMatcher(data interface{}, toMatch string) bool {
+
+	jwt, err := GetJWT(toMatch)
+	if err != nil {
+		log.Errorf("Error occurred while fetching jwt token %s", err.Error())
+		return false
+	}
+	fmt.Println(jwt)
```

Comment:
guess it was for debugging, could you remove it please?

Reply:
yup, done. I didn't want to run tests in debug mode again n again :p